pkg/system/phase2_creating.go

```diff
@@ -342,6 +347,11 @@ func (r *Reconciler) setDesiredCoreEnv(c *corev1.Container) {
 				c.Env[j].Value = "mongodb://" + r.NooBaaMongoDB.Name + "-0." + r.NooBaaMongoDB.Spec.ServiceName + "/nbcore"
 			}
 
+		case "NOOBAA_LOG_LEVEL":
+			if r.NooBaa.Spec.DebugLevel != 0 {
```

Comment:
what about the else case? if you never remove it then there is not way to disable this setting once it was done...
check out the other optional spec fields that have both a set and unset of an env.

Reply:
Removed the if completely as we have a default in the yaml (0)